README.md

```diff
@@ -20,7 +20,14 @@ yarn add @fullstory/browser
 
 ## Initialize the SDK
 
-Call the `init()` function as soon as you can in your website startup process. 
+Call the `init()` function with options as soon as you can in your website startup process.
+
+| Option | Required | Description |
+| ------ | -------- | ----------- |
+| orgId  | yes | Your FullStory Org ID |
+| debug | no | Enables FullStory recording debug messages; defaults to `false` |
+| iFrameOption | no | Controls recording of iFrames; read the security implications if using `RecordCrossDomainIFrame` in [help](https://help.fullstory.com/hc/en-us/articles/360020622514) |
```

Comment:
This is Great! Long overdue :) 

Suggestion: consider including the 2 values expected for iFrameOption here: `_fs_run_in_iframe` and `_fs_is_outer_script`

Reply:
This brings up an interesting decision.  We favor the enum because it conveys more semantics to the iFrame options.  If we think `RecordCrossDomainIFrame` is more understandable than `_fs_run_in_iframe`, perhaps we should just say to use it: `FullStory.IFrameOption.RecordCrossDomainIFrame`.  This would abstract away the underlying, opaque value to something the SDK provides.  Said differently, the SDK now provides the needed constants versus the developer having to deal with strings.

Reply:
I also updated it a bit to ensure the reader was clear that `RecordCrossDomainIFrame` applies to `_fs_run_in_iframe` as a security concern.